hooks/03-writable-paths.chroot

```diff
@@ -15,16 +15,10 @@ cat > /etc/system-image/writable-paths << __WRITABLE__
 # --------------------------------------------------------------------
 /etc/apparmor.d/cache                   auto                    persistent  none        none
 /etc/dbus-1/system.d                    auto                    persistent  transition  none
-/etc/default/swapfile                   auto                    persistent  transition  none
```

Comment:
Can you please (pretty please) split this patch into one-per-removal that justifies how we can drop each line.

Reply:
After discussing on IRC we will drop the whole file and then slowly add things back. Since the base is not designed for booting yet this is sufficient.